gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -67,7 +67,10 @@
             <div class="col-md-12">
               <div class="form-group">
                 <label translate for="folders">Share Folders With Device</label>
-                <p translate class="help-block">Select the folders to share with this device.</p>
+                <p class="help-block">
+                  <span translate>Select the folders to share with this device.</span>&emsp;
+                  <small><a href="#" ng-click="selectAllFolders()" translate>Select All</a> - <a href="#" ng-click="deSelectAllFolders()" translate>Deselect All</a></small>
```

Comment:
I'm not sure if you need the `href="#"`part, maybe you can test this.

Reply:
Actually no needed. I am going to test it. Thanx

Reply:
It's actually nice to have `href="#"` because it means you get the "hand" mouse cursor over the links. Otherwise we get just the normal text selection cursor that doesn't indicate an activity.

Reply:
Yeah, good point, I didn't think like that, thanks